packages/components-js/tests/vrt/playwright/specs/states/checkbox-wrapper.vrt.ts

```diff
@@ -72,14 +70,14 @@ const scenario = async (page: Page, theme: Theme, scheme?: PrefersColorScheme):
     prefersColorScheme: scheme,
   });
 
-  await forceHoverState(page, '.hover p-checkbox-wrapper input[type="checkbox"]');
-  await forceHoverState(page, '.hover .force-label > p-checkbox-wrapper >>> span');
+  await forceHoverState(page, '.hover p-checkbox-wrapper:not(.force-label) input[type="checkbox"]');
+  await forceHoverState(page, '.hover p-checkbox-wrapper.force-label >>> span');
   await forceHoverState(page, '.hover p-checkbox-wrapper span a');
   await forceFocusState(page, '.focus p-checkbox-wrapper input[type="checkbox"]');
   await forceFocusState(page, '.focus p-checkbox-wrapper span a');
-  await forceFocusHoverState(page, '.focus-hover p-checkbox-wrapper input[type="checkbox"]');
-  await forceFocusState(page, '.focus-hover .force-label > p-checkbox-wrapper input[type="checkbox"]');
-  await forceHoverState(page, '.focus-hover .force-label > p-checkbox-wrapper >>> span');
+  await forceFocusHoverState(page, '.focus-hover p-checkbox-wrapper:not(.force-label) input[type="checkbox"]');
+  await forceFocusState(page, '.focus-hover p-checkbox-wrapper.force-label input[type="checkbox"]');
```

Comment:
Isn't this redundant with `await forceFocusState(page, '.focus p-checkbox-wrapper input[type="checkbox"]');` from a few lines above?